src/test/java/ru/training/at/hw2/TestBaseClass.java

```diff
@@ -0,0 +1,23 @@
+package ru.training.at.hw2;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.asserts.SoftAssert;
+
+public class TestBaseClass {
+    public WebDriver webDriver;
+
+    @BeforeTest
+    public void beforeTest(){
+        System.setProperty("webdriver.chrome.driver", "C:\\chromedriver.exe");
```

Comment:
Why do you decide do not use `WebDriverManager` from boniagarcia for the managing of the driver?
Your test could be running only on Windows PC

Reply:
On the lecture was said that we can use this way too, so I decided use it. 
Resolved this comment and used WebDriverManager instead